lib/block_view_message.go

```diff
@@ -825,6 +856,76 @@ func (bav *UtxoView) _connectMessagingGroup(
 		}
 	}
 
+	// V3 Messages: MUTING and UNMUTING members
+	if blockHeight >= bav.Params.ForkHeights.DeSoV3MessagesMutingBlockHeight {
+		if existingEntry != nil && !existingEntry.isDeleted {
+			value, operationTypeExists := txn.ExtraData["OperationType"]
+			if operationTypeExists {
+				// inline helper method
+				contains := func(memberList []*MessagingGroupMember, member *MessagingGroupMember) bool {
```

Comment:
Since we're using `contains` only once, maybe we just write that logic directly in place where we were supposed to use `contains`?

Reply:
moved `contains` right above its usage, but keeping the function as it helps simplify logic